odoo/addons/base/res/res_country_view.xml

```diff
@@ -110,6 +110,9 @@
             <field name="model">res.country.state</field>
             <field name="arch" type="xml">
                 <form string="State">
+                    <div class="oe_button_box">
```

Comment:
Would not it be better to standardize the state registry, in the same way as the country registry? With this div it would be possible to add actions within this register to simplify the user experience. For example, I'm working on a module for searching postal codes. I would like to add a quick search by returning only the codes of this state, facilitating the registration of new codes. In the same way as the registration of cities for a certain state.